mm-go-irckit/mmservice.go

```diff
@@ -66,28 +67,24 @@ func login(u *User, toUser *User, args []string, service string) {
 			fmt.Println("login, starting logout")
 			err := u.logoutFromSlack()
 			if err != nil {
-				u.MsgUser(toUser, err.Error())
+				u.MsgUser(toUser, err.Error(), irc.ERROR)
 				return
 			}
 		}
 		if u.mc != nil {
 			err := u.logoutFromMattermost()
 			if err != nil {
-				u.MsgUser(toUser, err.Error())
+				u.MsgUser(toUser, err.Error(), irc.ERROR)
 				return
 			}
 		}
 		u.inprogress = true
 		defer func() { u.inprogress = false }()
 		u.sc, err = u.loginToSlack()
 		if err != nil {
-			u.MsgUser(toUser, err.Error())
+			u.MsgUser(toUser, err.Error(), irc.ERROR)
 			return
 		}
-		u.MsgUser(toUser, "login OK")
-		if u.Credentials != nil && u.Token != "" {
-			u.MsgUser(toUser, "token used: "+u.Token)
-		}
```

Comment:
I would like to keep those messages, what is your reasoning to remove those?

Reply:
I removed the login OK since there is an automatic disconnect from the server now anyways if the login is not OK.
lines 88-90 I'm not sure anymore, might have been an accident...